src/features/people/PersonPage/index.js

```diff
@@ -1,13 +1,75 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import {
+  fetchPersonDetails,
+  fetchPersonDetailsError,
+  getPersonId,
+  selectCast,
+  selectCrew,
+  selectDetails,
+  selectStatus
+} from "./personSlice";
 import { MainPersonTile } from "./MainPersonTile";
 import { MainHeader } from "../../../common/MainHeader";
 import { Container } from "../../../common/Container/styled";
+import { Loading } from "../../../common/Loading";
+import { ErrorPage } from "../../../common/ErrorPage";
+
 
 export const PersonPage = () => {
-    return (
-        <Container>
-            <MainPersonTile />
-            <MainHeader title="Movies - cast (4)" />
-            <MainHeader title="Movies - crew (4)" />
-        </Container>
-    )
-};
\ No newline at end of file
+  const dispatch = useDispatch();
+  const { personId } = useParams();
+  const details = useSelector((state) => {
+    const details = selectDetails(state);
+    console.log("Details:", details); // Dodaj ten console.log
+    return details;
+  });
+  const cast = useSelector((state) => {
+    const cast = selectCast(state);
+    console.log("Cast:", cast); // Dodaj ten console.log
+    return cast;
+  });
+  const crew = useSelector((state) => {
+    const crew = selectCrew(state);
+    console.log("Crew:", crew); // Dodaj ten console.log
+    return crew;
```

Comment:
The code looks nice, in the final version I think console.logs will be removed? :)

Reply:
same same :)